src/Control/Lens/Internal/TH.hs

```diff
@@ -59,11 +59,7 @@ conAppsT :: Name -> [Type] -> Type
 conAppsT conName = foldl AppT (ConT conName)
 
 fromSet :: (k -> v) -> Set.Set k -> Map.Map k v
```

Comment:
Similarly, is there a good reason not to remove this function and replace its use sites with `Map.fromSet`?

Reply:
Done.